src/components/SubscriptAvatar.js

```diff
@@ -0,0 +1,61 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import Tooltip from './Tooltip';
+import themeColors from '../styles/themes/default';
+import Avatar from './Avatar';
+import CONST from '../CONST';
+import * as StyleUtils from '../styles/StyleUtils';
+
+const propTypes = {
+    /** Avatar URL or icon */
+    mainAvatar: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
+
+    /** Subscript avatar URL or icon */
+    secondaryAvatar: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
+
+    /** Tooltip for the main avatar */
+    mainTooltip: PropTypes.string,
+
+    /** Tooltip for the subscript avatar */
+    secondaryTooltip: PropTypes.string,
+
+    /** Set the size of avatars */
+    mode: PropTypes.oneOf(_.values(CONST.OPTION_MODE)),
```

Comment:
NAB but in your follow-up I would recommend changing this prop to `size` to match the other avatar components

Reply:
Sure thing, also I will add a Storybook stories of SubscriptAvatar

Reply:
I addressed this change here https://github.com/Expensify/App/pull/8144